src/main/kotlin/uk/gov/dluhc/printapi/service/CertificateDataRetentionService.kt

```diff
@@ -37,4 +39,16 @@ class CertificateDataRetentionService(
                 ?: logger.error { "Certificate with sourceType = $sourceType and sourceReference = $sourceReference not found" }
         }
     }
+
+    @Transactional
+    fun removeInitialRetentionPeriodData(sourceType: SourceType) {
+        logger.info { "Finding certificates to remove initial retention period data from" }
```

Comment:
Whilst you are logging these, it will be worth adding sourceType to both log statements in this method

Reply:
I've added it to this message, rather than each one below.